Expo1900/Expo1900/Controller/ListViewController.swift

```diff
@@ -30,19 +45,25 @@ extension ListViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: ItemTableViewCell.identifier, for: indexPath)
         guard let itemCell = cell as? ItemTableViewCell else { return cell }
-        itemCell.itemImageView.image = UIImage(named: items[safe: indexPath.row].imageName)
-        itemCell.titleLabel.text = items[safe: indexPath.row].name
-        itemCell.shortDescriptionLabel.text = items[safe: indexPath.row].shortDescription
+        guard let item = items[safe: indexPath.row] else { return cell }
+        itemCell.displayWith(item: item)
+        itemCell.changeItemStackViewSetting()
+        
         return itemCell
     }
 }
 
 extension ListViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         tableView.deselectRow(at: indexPath, animated: true)
-        guard let itemVC = storyboard?.instantiateViewController(withIdentifier: ItemViewController.identifier) as? ItemViewController else { return }
+        guard let item = items[safe: indexPath.row] else { return }
+        let itemVC = storyboard?
+            .instantiateViewController(identifier: ItemViewController.identifier,
+                                       creator: { coder -> ItemViewController in
+                return .init(coder, item) ?? ItemViewController(item: item)
+            })
```

Comment:
앗! 요기서 위에 코멘트한 생성자가 쓰이는군요..!
지금은 `.init(coder, item)`가 실패하면 `fatalError`가 호출되도록 처리한 것과 다름없을 것 같습니다 🤔 
```suggestion
        let itemVC = storyboard?
            .instantiateViewController(identifier: ItemViewController.identifier,
                                       creator: { coder -> ItemViewController? in
                return .init(coder, item)
            })
```
요렇게 바꾸면 혹시나 `.init(coder, item)`가 실패하더라도 앱이 종료되는 상황은 막을 수 있을 것 같아요!

Reply:
네넵 이부분을 수정해 주었습니다!! fatalError 를 쓰지 말았어야 하는 것을 잘 몰랐었네요ㅠㅠ